roles/science/tasks/mrtrix.yml

```diff
@@ -4,30 +4,19 @@
     path: "{{ item }}"
     state: directory
   loop:
-    - /opt/quarantine/software/mrtrix3/3.0.4
+    - /opt/quarantine/software/mrtrix3/{{ mrtrix3_version }}
     - /opt/quarantine/software/mrtrix3tissue/5.2.9
 
-- name: install mrtrix3
-  shell: micromamba create --quiet --yes -c mrtrix3 -c conda-forge -p /opt/quarantine/software/mrtrix3/3.0.4/mrtrix3 mrtrix3=3.0.4
-  args:
-    creates: /opt/quarantine/software/mrtrix3/3.0.4/mrtrix3/bin/mrcalc
-
-- name: Install module
-  template:
-    src: templates/conda_module.j2
-    dest: /opt/quarantine/software/mrtrix3/3.0.4/module
-  vars:
-    modulename: mrtrix3
-  notify: link modules
 
-- name: Install mtrix3tissue dependencies
+- name: Install mtrix3 dependencies
```

Comment:
typo

Reply:
I also noticed the typo in the branch name too late hehe